lectures/cons_smooth.md

```diff
@@ -147,11 +147,11 @@ We create a Python **namedtuple** to store these parameters with default values.
 ConsumptionSmoothing = namedtuple("ConsumptionSmoothing", 
                         ["R", "g1", "g2", "β_seq", "T"])
 
-def creat_cs_model(R=1.05, g1=1, g2=1/2, T=65):
+def create_consumption_smoothing_model(R=1.05, g1=1, g2=1/2, T=65):
```

Comment:
thanks @longye-tian. 

The build is failing and I suspect the calling location needs to be updated to this new function name as well.

Reply:
Thank you @mmcky !

I just changed the function name at the calling location. :)

Reply:
thanks @longye-tian -- you will see the preview is now generating. 👍

Reply:
So glad to contribute !!